src/evse_monitor.cpp

```diff
@@ -254,8 +254,16 @@ void EvseMonitor::evseBoot(const char *firmware)
     _heartbeat = RAPI_RESPONSE_OK == ret;
   });
 
+  // ESP32_WiFi handle OpenEvse state, so let first put it to disabled before unlocking
+  if (config_pause_uses_disabled()) {
```

Comment:
Not convinsed this is the correct place for this, this is only useful if the target state is disabled, if the target state is active, this will cause an unnecessary switch to sleep and then back to active/charging. In any case the monitor should just get the current state and let the manager deside what to do. Also FYI the EVSE manager has a local copy of the config. Another thing to consider is that we are going to have to extend the 'pause' behaviour to handle some of the weird things various vehicles do, so having this logic in multiple places will make this harder.

Reply:
Only the latest firmware has the boot lock so still have to support firmwares with out bootlock support. Probably what needs to happen is to unlock the bootlock in the main loop after the state has been evaluated and set the first time

Reply:
I've changed the bootlock method with latest commit https://github.com/OpenEVSE/ESP32_WiFi_V4.x/pull/618/commits/c1fba7de64e362394f56d58d29ef3f65434ae389

Reply:
So i've pushed to your openevse_lib the missing vflags : https://github.com/jeremypoulter/OpenEVSE_Lib/pull/9
Now with latest [commit ](https://github.com/OpenEVSE/ESP32_WiFi_V4.x/pull/618/commits/e27c9a3f424bde4a75eca0e64ca8ca4a9c892123) it tests correctly the bootlock condition